docs/api.rst

```diff
@@ -12122,3 +12119,50 @@ Custom Assets
    :statuscode 200: Custom asset types retrieved successfully.
    :statuscode 409: No user is currently logged in. Check error message.
    :statuscode 500: Internal rotki error.
+
+
+Events Details
+================
+
+.. http:get:: /api/(version)/history/events/details
+
+   Doing a GET on this endpoint will return the details of a history event.
+
+   **Example Request**:
+
+   .. http:example:: curl wget httpie python-requests
+
+      POST /api/1/assets/custom HTTP/1.1
+      Host: localhost:5042
+      Content-Type: application/json;charset=UTF-8
+
+      {"identifier": 137}
+
+   :reqjson int identifier: The identifier of the event to be queried.
+
+   **Example Response**:
+
+   .. sourcecode:: http
+
+      HTTP/1.1 200 OK
+      Content-Type: application/json
+
+      {
+          "result": {
+              "sub-swaps": [
+                  {"from_amount": "100.0", "to_amount": "0.084", "from_asset": A_SUSHI.identifier, "to_asset": A_ETH.identifier},  # noqa: E501
+                  {"from_amount": "0.084", "to_amount": "98.2", "from_asset": A_ETH.identifier, "to_asset": A_USDT.identifier},  # noqa: E501
```

Comment:
This is invalid json copy pasta Use the correct values and make sure that there is no trailing commas. Run make docs.

```suggestion
                  {"from_amount": "100.0", "to_amount": "0.084", "from_asset": A_SUSHI.identifier, "to_asset": A_ETH.identifier},  # noqa: E501
                  {"from_amount": "0.084", "to_amount": "98.2", "from_asset": A_ETH.identifier, "to_asset": A_USDT.identifier},  # noqa: E501
```